pom.xml

```diff
@@ -87,10 +87,6 @@
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
             <version>2.2.0</version>
         </dependency>
-        <dependency>
```

Comment:
we need this dependency. I guess you disabled it inadvertently while merging branches

Reply:
Added